app/src/main/res/layout/nav_tab.xml

```diff
@@ -0,0 +1,25 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/nav_tab"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:gravity="center_horizontal">
+
+    <ImageView
+        android:id="@+id/navIcon"
+        android:layout_width="55dp"
+        android:layout_height="48dp"
+        android:layout_marginBottom="5dp"
+        android:scaleType="centerInside" />
+
+    <TextView
+        android:id="@+id/nav_label"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:fontFamily="serif-monospace"
+        android:shadowColor="@android:color/black"
+        android:textColor="@android:color/darker_gray"
+        />
```

Comment:
this should be at the end of the las line

Reply:
Oh, I forgot to delete this file. I'm not actually using it. I've created it trying something.

Reply:
File deleted!